stumpy/scrump.py

```diff
@@ -158,25 +204,43 @@ def _compute_diagonal(
     I : ndarray
         Matrix profile indices
 
+    thread_idx : int
+
```

Comment:
Can we add the description to this argument and move `thread_idx` above `P` so that it is consistent with the function arguments list?

Reply:
Sure, what description do you suggest?